src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -126,11 +128,12 @@ function ReportActionItemSingle({
     } else {
         secondaryAvatar = {name: '', source: '', type: 'avatar'};
     }
+
     const icon = {
-        source: avatarSource,
+        source: avatarSource ?? FallbackAvatar,
         type: isWorkspaceActor ? CONST.ICON_TYPE_WORKSPACE : CONST.ICON_TYPE_AVATAR,
         name: primaryDisplayName ?? '',
-        id: isWorkspaceActor ? '' : actorAccountID,
+        id: avatarAccountId,
```

Comment:
You missed to set `avatarAccountId` to `undefined` if `isWorkspaceActor`

Reply:
@s77rt but last time you told me to change it here to this specific code: https://github.com/Expensify/App/pull/39229#discussion_r1566306188

so which version do you think it should be?

Reply:
I meant to set that here
https://github.com/Expensify/App/blob/445a70e923134e4a19444e84b61e595fff0b6af2/src/pages/home/report/ReportActionItemSingle.tsx#L92-L96

below `avatarSource = ...` add `avatarAccountId = undefined`

Reply:
okay understood 👍